graphql-bal-service/src/main/java/module-info.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+module io.ballerina.graphql.schema {
+    requires io.ballerina.lang;
+    requires io.ballerina.parser;
+    requires io.ballerina.tools.api;
+    requires org.apache.commons.io;
+    requires io.ballerina.stdlib.graphql.commons;
+    exports io.ballerina.graphql.schema.diagnostic;
+    exports io.ballerina.graphql.schema.exception;
+    exports io.ballerina.graphql.schema.generator;
+    exports io.ballerina.graphql.schema;
```

Comment:
What is the reason for these exports?

Reply:
These are used in the `graphql-cli` module

Reply:
Then shall we export these to those exact modules instead of exporting them publicly?

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/pull/118/commits/152c46cc2b96601de25b9d2d7acae59f647699b6

Reply:
Removed this due to a warning.